Project/backend/codebase/graph_creator/graph_creator_main.py

```diff
@@ -82,6 +82,7 @@ def create_and_store_graph(uuid, entities_and_relations, chunks):
 
     # combine knowledge graph pieces
     # combined = graph_handler.connect_with_chunk_proximity(df_e_and_r)
+    # combined['chunk_id'] = '1'
```

Comment:
you can remove if not needed

Reply:
It's needed if you want to use the other way of connecting graphs. If you uncomment both lines you can use it.